indigo_api/models.py

```diff
@@ -1201,6 +1201,7 @@ def assign_to(self, assignee, assigned_by):
         """ Assign this task to assignee (may be None)
         """
         self.assigned_to = assignee
+        self.save()
         if assigned_by == self.assigned_to:
```

Comment:
should https://github.com/laws-africa/indigo/blob/master/indigo_app/views/tasks.py#L301 move here 
and https://github.com/laws-africa/indigo/blob/master/indigo_app/views/tasks.py#L302 come out?